packages/sil-governance/src/components/Header/index.tsx

```diff
@@ -1,19 +1,48 @@
 import { Address } from "@iov/bcp";
 import { Electorate } from "@iov/bns";
 import { Block, CircleImage, Image, Typography } from "medulas-react-components";
-import React from "react";
+import React, { useMemo } from "react";
 
 import iovLogo from "../../assets/iov-logo-title.svg";
 import userIcon from "../../assets/user.svg";
+import { history } from "../../routes";
+import { SHOW_ELECTORATE_ROUTE } from "../../routes/paths";
 
 export const HEADER_HTML_ID = "header";
 
+const navigateToElectorate = (id: number): void => {
+  const target = `${SHOW_ELECTORATE_ROUTE}/${id}`;
+  if (history.location.pathname !== target) {
+    history.push(target);
+  }
+};
+
 interface Props {
   readonly address: Address;
   readonly electorates: readonly Electorate[];
 }
 
 const Header = ({ address, electorates }: Props): JSX.Element => {
+  const ElectorateLinks = useMemo(
+    () => (): JSX.Element => (
+      <React.Fragment>
+        {electorates.map((e, index) => (
+          <Block display="inline" key={e.id}>
+            <Typography inline link variant="body2" onClick={() => navigateToElectorate(e.id)}>
```

Comment:
By default, a Link should be created using the `<Link` component, which can be combinded with Typography like

```tsx
    <Link to="https://iov.one">
      <Typography weight="light" link color="primary">
        Go to IOV
      </Typography>
    </Link>
```

or

```tsx
          <Typography link={!!blockExplorerUrl} inline>
            {blockExplorerUrl ? (
              <Link to={blockExplorerUrl}>{`${recipientShort}`}</Link>
            ) : (
              `${recipientShort}`
            )}
          </Typography>
```

this avoids all the manual history manipulation, which should be covered by the router

Reply:
I'm getting `Uncaught Error: Invariant failed: You should not use <Link> outside a <Router>` when using a partial url as the `to` parameter. So I can set `to` to `https://www.google.com` but I can't set it to `/create-proposal` or whatever.

However in Neuma i can see the following code in `UnlockControls.tsx` without issue:
```typescript
<Link to={RESTORE_WALLET}>
    <Typography variant="subtitle2" color="primary" link inline>
        Restore wallet
    </Typography>
</Link>
```

And in `paths.ts`:
```typescript
export const RESTORE_WALLET = "/restore-wallet";
```

Reply:
Where exactly do you get that exception? In the application or the storybook? The sil application has a `<Router` in the `<Routes />` component. However, the storybook misses a `<Router`, which might be a problem. We might need a solution like `SanesStorybook` for the governance app as well.

Reply:
The app crashed as soon as it tried to render a `<Link>`. I've solved this for now by using `react-router-dom`'s `<Link>` instead of medulas'.

Reply:
Hard to tell without the stack tract. Maybe https://github.com/iov-one/ponferrada/pull/809 helps. Anyways, there is nothing wrong with using `<Link>` from `react-router-dom` directly